tests/common.py

```diff
@@ -22,6 +22,13 @@ def _reset_schema():
     rest.db.engine.execute('drop schema if exists public cascade;')
     rest.db.engine.execute('drop schema if exists disclosure cascade;')
     rest.db.engine.execute('create schema public;')
+    rest.db.engine.execute('create schema disclosure;')
+
+
+def _reset_schema_for_integration():
+    rest.db.engine.execute('drop schema if exists public cascade;')
+    rest.db.engine.execute('drop schema if exists disclosure cascade;')
+    rest.db.engine.execute('create schema public;')
```

Comment:
This PR had a couple of huge headaches, this is one of them.  I'll do my best to explain what I think is happening.  For the integration tests, we actually use the `subset.dump` to populate `cfdm_unit_test` using `pg_restore`.  For the public schema this worked fine.  However with `disclosure` this was causing an error.  And it seems that, and I'm postulating here, that pg_dump doesn't dump the public schema explicity.  But any other custom schema it does drump a `create schema` statement in the binary dump.  So when we attempt to run `create schema disclosure` from within the test set-up, it causes an error, stating that the schema already exists.  So this was a solution, maybe not the best.  The only thing I could think to do is filter out the `create schema disclosure` statement in the dump file, but that's an extra step, also there is no way to exclude these statements being exported via command line, as I hoped.